# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/lbnames.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./cppu/inc/uno/lbnames.h	2009-08-06 08:01:05.000000000 +0000
+++ ./cppu/inc/uno/lbnames.h	2010-08-08 17:09:24.323893660 +0000
@@ -62,12 +62,12 @@
 #error "Tested gcc 2 versions are 2.91 and 2.95.  Patch uno/lbnames.h to try your gcc 2 version."
 #elif (__GNUC__ == 3 && __GNUC_MINOR__ <= 4)
 #define __CPPU_ENV gcc3
-#elif (__GNUC__ == 4 && __GNUC_MINOR__ <= 4)
+#elif (__GNUC__ == 4 && __GNUC_MINOR__ <= 9)
 #define __CPPU_ENV gcc3
 #elif (__GNUC__ == 3)
 #error "Tested gcc 3 version is <= 3.4.  Patch uno/lbnames.h to try your gcc 3 version."
 #else
-#error "Supported gcc majors are 2 , 3 and 4 <= 4.4.  Unsupported gcc major version."
+#error "Supported gcc majors are 2 , 3 and 4 <= 4.x.  Unsupported gcc major version."
 #endif /* defined (__GNUC__) */
 #endif /* defined (_MSC_VER) */
 
